app/server/lib/ICreate.ts

```diff
@@ -117,5 +119,11 @@ export function makeSimpleCreator(opts: {
       elements.push(getThemeBackgroundSnippet());
       return elements.join('\n');
     },
+    getStorageOptions(name: string) {
+      for (const s of storage || []) {
+        console.log({s, name});
+        if (s.name === name) { return s; }
+      }
```

Comment:
```suggestion
      return storage?.find(s => s.name === name);
```

Is the `console.log()` for debugging?